code/controllers/subsystem/processing/modular_computers.dm

```diff
@@ -1,6 +1,9 @@
-SUBSYSTEM_DEF(modular_computers)
+PROCESSING_SUBSYSTEM_DEF(modular_computers)
```

Comment:
I don't understand the reasoning behind processing here, do we really want it tied like this?

Reply:
Computers no longer get processed under SSobj since they are no longer `/obj/item/modular_computer`s, but I still needed processing functionality